src/main/java/frc/robot/subsystems/conveyor/Conveyor.java

```diff
@@ -134,6 +134,9 @@ public int getColorSensorProximity() {
         return colorSensor.getProximityValue();
     }
 
+    public DriverStation.Alliance getColor() {

```

Comment:
I really hate to write this type of comments, but someone has to do that (unfortunately, it's me).
documentation